cli/broadlink_cli

```diff
@@ -217,6 +217,8 @@ if args.rfscanlearn:
     data = None
     timeout = 20
 
+    time.sleep(10)
```

Comment:
I understand what you are doing here. You were getting a ReadError, right? This will be resolved with [this PR](https://github.com/mjg59/python-broadlink/pull/365/files), we don't need to wait 10 secs.

Reply:
Great! glad to hear it, I can remove this change